### ISSUE TITLE
Quotation marks in code blocks and fragments should not be angled

### ISSUE THREAD

**Issue by evildmp** [django/djangoproject.com#275]
See https://docs.djangoproject.com/en/1.7/ref/templates/builtins/#url just for example.

    {% url 'path.to.some_view' v1 v2 %}

or https://docs.djangoproject.com/en/1.7/ref/templates/builtins/#cut:

    {{ value|cut:" " }}

These ought to be plain, straight marks, but they are both angled closing marks.

**Comment by jezdez** [comment]
Hm, that is the default quote character of the Inconsolata font (see http://typecast.com/preview/google/Inconsolata) and won't be a problem when copy-pasting. This is *not* a case of the quotes being turned into pretty quotes accidentally. Can you elaborate why you don't want them to be angled?

**Comment by timgraham** [comment]
Maybe it's just something we have to get used to, but I have never seen this in a monospace font.

**Comment by jezdez** [comment]
Yeah, so question is whether we can jump to another font that is also nice but doesn't have this misleading character? @andrevvm any hint?

**Comment by andrevvm** [comment]
Good call, I never noticed that the quotes were angled with Inconsolata.. can't unsee. I have a branch using Fira Mono (http://www.google.com/fonts/specimen/Fira+Mono). It's a bit heavier in general, but still a nice fit. Let me know what you guys think.

**Inconsolata**

![image](https://cloud.githubusercontent.com/assets/742229/5636048/c4685f74-95ab-11e4-8e7c-11d3dbabd498.png)

**Fira Mono**

![image](https://cloud.githubusercontent.com/assets/742229/5636053/d1ce46ba-95ab-11e4-9b18-81cb61bd1580.png)

Pull request: https://github.com/django/djangoproject.com/pull/285

**Comment by evildmp** [comment]
Looks fine to me, I don't think the extra heaviness looks out of place.
